### PR TITLE
Handle form

### PR DESCRIPTION
**Pull request by Ilaiwi**

### CODE REVIEW

**Comment by paramsingh** [comment]
I think removing the pyc files and the visual studio settings would be a great idea.

File: app.py [hunk 1]
```diff
@@ -21,11 +25,17 @@ def donate():
...
         return render_template('donate.html')
     elif request.method == 'POST':
-        donater_name = request.form.get('donater_name')
-        amount = request.form.get('amount')
+        first=request.form.get('first')
+        last=request.form.get('last')
+        cc=request.form.get('cc')
+        email=request.form.get('email')
+        amount= request.form.get('amount')
+        donater_name=first+last
```
Comment (kenleezle) [inline app.py:34]: 👍

File: app.py [hunk 2]
```diff
@@ -21,11 +25,17 @@ def donate():
...
         sent_txid = bdb_donate(blockchain_db, user, donater_name, amount)
-        return "Transaction {} sent".format(sent_txid)
-
+        userId = addDonation(mongodb,first,last,email,sent_txid)
```
Comment (kenleezle) [inline app.py]: 👍

**Review by kenleezle — APPROVED** [review]

**Comment by paramsingh** [comment]
LGTM, now that we've fixed the conflicts and added a gitignore. :+1:

**Pull request merged**